Strata/Languages/Boogie/Examples/Examples.lean

```diff
@@ -21,7 +21,6 @@ import Strata.Languages.Boogie.Examples.OldExpressions
 import Strata.Languages.Boogie.Examples.PrecedenceCheck
 import Strata.Languages.Boogie.Examples.ProcedureCall
 import Strata.Languages.Boogie.Examples.Quantifiers
-import Strata.Languages.Boogie.Examples.QuantifiersWithTypeAliases
```

Comment:
I noticed this file was removed from Examples.

What's the plan for this file?  I think we should remove files that aren't building unless there's an issue tracking it and clear plan to fix soon.

Reply:
Ah, I didn't mean to remove it. I added it, and then removed it because I wasn't sure it was actually working before. But that reversal of my addition undid your addition when I did the merge. I'll re-add it.